app/login/page.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @next/next/no-img-element */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+'use client';
+
+import { IDKitWidget, VerificationLevel, useIDKit } from '@worldcoin/idkit';
+import type { ISuccessResult } from '@worldcoin/idkit';
+
+
+export default function LoginPage() {
+  const { setOpen } = useIDKit();
+
+  const handleSuccess = (result: ISuccessResult) => {
+    window.alert(
+      'Successfully verified! Your nullifier hash is: ' + result.nullifier_hash
+    );
+  };
+
+  return (
+    <div className="flex h-screen">
+      {/* Left Section */}
+      <div className="flex flex-col justify-center items-center w-1/2 bg-black text-white">
```

Comment:
You did not use the specified background in the figma file

Reply:
Ok will push the update now